include/veriblock/fmt.hpp

```diff
@@ -0,0 +1,24 @@
+#ifndef ALT_INTEGRATION_INCLUDE_VERIBLOCK_FMT_HPP_
+#define ALT_INTEGRATION_INCLUDE_VERIBLOCK_FMT_HPP_
+
+#include <veriblock/third_party/tinyfmt.hpp>
+
+/**
+ * A facade for a third party formatting library.
+ *
+ * Now it is tinyfmt, but can be any.
+ */
+
+namespace VeriBlock {
+
+/**
+ * A facade for a printf-like formatting.
+ */
+template <typename... Args>
+std::string format(const char* fmt, const Args&... args) {
```

Comment:
Why do we need this? Ain't standard printf good enough?

Reply:
pay attention that this function does not **print** anything. It works as printf which writes into std::string 
```
std::string f = printf("%d", 50);
```